### PR TITLE
Implemented components to the subscription page.

### PR DESCRIPTION
**Pull request by MarcelloAragoni**
**Trello Task**: https://trello.com/c/DzVHcHwu/6-s

## Description
Implemented components and the styles of the subscription page.

## Screenshot
image

### CODE REVIEW

File: src/pages/Subscription/Subscription.jsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
+import Button from "../../Components/Button/Button";
+import Input from "../../Components/Input/input";
+import "./Subscription.scss";
+
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.jsx:4]:
```suggestion
import Input from "../../Components/Input/input";

import "./Subscription.scss";
```

File: src/pages/Subscription/Subscription.jsx [hunk 2]
```diff
@@ -1,14 +1,18 @@
+import Button from "../../Components/Button/Button";
+import Input from "../../Components/Input/input";
+import "./Subscription.scss";
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.jsx]:
```suggestion
import Button from "../../Components/Button/Button";
import Input from "../../Components/Input/input";

import "./Subscription.scss";
```

File: src/pages/Subscription/Subscription.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.info {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 0 auto;
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.scss]: There is a website that helps us to organizer the order of the CSS order, take a look at it: https://9elements.com/css-rule-order/

isn't mandatory, but it's a nice plus+.

File: src/pages/Subscription/Subscription.scss [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 0 auto;
+
+  h1 {
+    font-size: 30px;
+    margin-bottom: 10px;
+
+    &::after {
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.scss]: Clever ⭐

File: src/pages/Subscription/Subscription.scss [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      content: ".";
+      color: blue;
+    }
+  }
+
+  p {
+    font-size: 40px;
+    margin: 10px 0px !important;
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.scss:17]: Do you need this `!important`?

File: src/pages/Subscription/Subscription.scss [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    color: #919191;
+
+    &::after {
+      content: "soon!";
+      font-weight: bold;
+      color: black;
+    }
+  }
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.scss]: In that case, I think it's better to put it directly on HTML than style it using a CSS class.
If someone else is looking at your page, it's hard to figure out where this `soon` is coming from.

What do you think?

File: src/pages/Subscription/Subscription.scss [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+  >p+p {
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.scss]:
```suggestion
  > p + p {

```

File: src/pages/Subscription/Subscription.scss [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    font-size: 16px;
+    color: black;
+    margin: 0px 0px 10px 0px !important;
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.scss]: Same here https://github.com/MarcelloAragoni/ping/pull/4/files#r840676857

File: src/pages/Subscription/Subscription.scss [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+  }
+
+}
+
+.subscription {
+  display: flex;
+  margin: 0 auto;
+  justify-content: center;
+
+  *+* {
```
Comment (ralfting) [inline src/pages/Subscription/Subscription.scss]:
```suggestion
  * + *  {

```

**Comment by ralfting** [comment]
The text " Subscribe and get notified" need a little more margin-bottom (more space between the text and the input/button component)

File: src/pages/Subscription/Subscription.scss (hunk 5 above)
Comment (MarcelloAragoni) [reply src/pages/Subscription/Subscription.scss:17]: yes, i tryed without it and doesn't work.

File: src/pages/Subscription/Subscription.scss (hunk 4 above)
Comment (MarcelloAragoni) [reply src/pages/Subscription/Subscription.scss]: thanks

File: src/pages/Subscription/Subscription.scss (hunk 6 above)
Comment (MarcelloAragoni) [reply src/pages/Subscription/Subscription.scss]: so, is best to put back the "soon" on the html, and remove this content?

**Review by ralfting — APPROVED** [review]

**Pull request merged**